src/include/distributed/multi_physical_planner.h

```diff
@@ -179,7 +179,7 @@ typedef struct Task
 	bool upsertQuery;              /* only applies to modify tasks */
 	char replicationModel;         /* only applies to modify tasks */
 
-	bool insertSelectQuery;
+	bool modifyWithSubquery;       /* true for multi-shard INSERT/UPDATE/DELETE queries */
```

Comment:
I know we haven't added very descriptive comments in this struct, but could we add one above saying what we use this for?